src/app.ts

```diff
@@ -1,6 +1,29 @@
 import "./components/maincontroller";
+import { CastReceiverContext, PlayerManager } from "chromecast-caf-receiver/cast.framework";
 
-let senders = cast.framework.CastReceiverContext.getSenders();
+declare global {
+    var $scope;
+
+    interface Window {
+        castReceiverContext : CastReceiverContext,
+        mediaManager : PlayerManager,
+        mediaElement,
+        VolumeInfo,
+        playlist,
+        currentPlaylistIndex,
+        hasReportedCapabilities,
+        DefaultMaxBitrate
+        MaxBitrate,
+        commandHandler,
+        deviceInfo,
+        reportEventType,
+        senderId,
+        subtitleAppearance,
+        repeatMode,
+    }
+}
+
+let senders = CastReceiverContext.getInstance().getSenders();
```

Comment:
This was reported as not working, was it fine when you tested? I applied it blindly since I don't own a cast device.

Reply:
By "this" do you mean:
`let senders = cast.framework.CastReceiverContext.getSenders();`
Because it's been changed to:
`let senders = CastReceiverContext.getInstance().getSenders();`
😀

Reply:
Ah, I see that now :laughing: that would be the one!

Reply:
Oh I just realized you had fixed this here, after submitting #48.